### PR TITLE
Add privacy policy to sign up page.

### PR DESCRIPTION
**Pull request by Tiger-Cross**

### CODE REVIEW

File: src/components/pages/SignUpPage.js [hunk 1]
```diff
@@ -3,10 +3,7 @@ import CardLayout from '../CardLayout';
 import SignUpForm from '../forms/SignUpForm/SignUpForm';
 
 export default class SignUpPage extends React.Component {
-  state = {
-    // forms: [<Spinner animation="border" />]
-    // forms: placeholderForms
-  };
+  state = {};
 
   // TODO: TIGER Privacy and Data security form.
```
Comment (wburr10) [inline src/components/pages/SignUpPage.js]: Should this TODO be here?

Comment (Tiger-Cross) [reply src/components/pages/SignUpPage.js]: Nope, I'll remove it today

File: src/components/forms/PrivacyPolicy/PrivacyPolicy.js [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+                  which includes but is not limited to (name, email and contact
+                  number).
+                </Card.Text>
+                <Card.Text>
+                  We use this information to operate, maintain, and provide to
+                  you the features and functionality of the Service. We do not
+                  sell, trade or otherwise transfer this information with any
+                  external third party organisations, however, in signing up it
+                  will be shared with the City Harvest Volunteer coordinator, as
+                  it is required for them to approve you as a new Volunteer. We
```
Comment (wburr10) [inline src/components/forms/PrivacyPolicy/PrivacyPolicy.js]: If we are doing invite by emails for sign-up, then the coordinator will not need to approve them as a new volunteer.

File: src/components/forms/PrivacyPolicy/PrivacyPolicy.js [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+                  also use this data to allow the Volunteer coordinator to
+                  contact you in case of any emergencies or potential shifts and
+                  you will be able to select your contact preferences after
+                  being accepted by the coordinator.
```
Comment (wburr10) [inline src/components/forms/PrivacyPolicy/PrivacyPolicy.js]: Same as above. If the invite-link code is merged then the coordinator need not 'accept' the volunteer if that makes sense

File: src/components/forms/PrivacyPolicy/PrivacyPolicy.js [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+                </Card.Text>
+                <Card.Text>
+                  Please note: The Charity may also ask additional questions as
```
Comment (wburr10) [inline src/components/forms/PrivacyPolicy/PrivacyPolicy.js]: As we aren't doing additional questions right now, this section may not need to be here

File: src/components/forms/PrivacyPolicy/PrivacyPolicy.js (hunk 3 above)
Comment (Tiger-Cross) [reply src/components/forms/PrivacyPolicy/PrivacyPolicy.js]: Oh yeah sorry man, I forgot to change this for the latest developments. I'll quickly fix that

File: src/components/PrivacyPolicy/PrivacyPolicy.js [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+                  We collect the routine information in the sign up form above
+                  which includes but is not limited to (name, email and contact
+                  number).
+                </Card.Text>
+                <Card.Text>
+                  We use this information to operate, maintain, and provide to
+                  you the features and functionality of the Service. We do not
+                  sell, trade or otherwise transfer this information with any
+                  external third party organisations, however, in signing up it
+                  will be shared with the City Harvest Volunteer coordinator, as
```
Comment (arjunsinghrana) [inline src/components/PrivacyPolicy/PrivacyPolicy.js]: Can we simplify this to just "City Harvest"

File: src/components/PrivacyPolicy/PrivacyPolicy.js [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+                  we use this data to allow the coordinator to contact you in
+                  case of any emergencies or potential shifts. You will be able
```
Comment (arjunsinghrana) [inline src/components/PrivacyPolicy/PrivacyPolicy.js]: Can we generalise this to "City Harvest business/legitimate interests that include but are not limited to ..."

**Pull request merged**